setup/base_external_dbsource_mysql/setup.py

```diff
@@ -2,5 +2,11 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'MySQLdb': 'mysqlclient',
```

Comment:
The version is fixed in https://github.com/OCA/server-backend/blob/56fba17abc90466e9d9f31dfaa5578940cca7083/requirements.txt#L2 so I think the module might have some issues if we simply install the latest version.
I have proposed https://github.com/hbrunn/server-backend/pull/3 to pin the version can you have a look and maybe merge it?

Reply:
thanks